src/wp-includes/formatting.php

```diff
@@ -2035,7 +2035,16 @@ function sanitize_file_name( $filename ) {
 	}
 
 	if ( $utf8_pcre ) {
-		$filename = preg_replace( "#\x{00a0}#siu", ' ', $filename );
+		/**
```

Comment:
This is a code/logic comment, not a docblock. I believe it should use the `/*` block starter, not the phpdoc `/**`
```suggestion
		/*
```

Reply:
If we do this we lose the likability of the `@see` and other PHPDoc tokens